pinot-plugins/pinot-stream-ingestion/pinot-kafka-2.0/src/main/java/org/apache/pinot/plugin/stream/kafka20/KafkaConsumerFactory.java

```diff
@@ -25,18 +25,22 @@
 
 public class KafkaConsumerFactory extends StreamConsumerFactory {
 
+  private static final String _PARTITION_LEVEL_CONSUMER = ".partition_level_consumer";
+  private static final String _PARTITION_METADATA_PROVIDER = ".partition_metadata_provider";
+  private static final String _STREAM_METADATA_PROVIDER = ".stream_metadata_provider";
+
   @Override
   public PartitionLevelConsumer createPartitionLevelConsumer(String clientId, int partition) {
-    return new KafkaPartitionLevelConsumer(clientId, _streamConfig, partition);
+    return new KafkaPartitionLevelConsumer(clientId + _PARTITION_LEVEL_CONSUMER, _streamConfig, partition);
```

Comment:
Since Kafka puts the clientId in its metrics, this might change metrics for people who use them right now.

cc: @mcvsubbu